DEFAULT_CONFIG.json5

```diff
@@ -475,6 +475,12 @@
             /// The maximum time limit (in ms) a message should be retained for batching when back-pressure happens.
             time_limit: 1,
           },
+          allocation: {
```

Comment:
Is there a result that shows that the first allocation has a sensible effect, which would justify the need of this configuration addition?